apps/docs/src/docs/components/demo/PopoverTriggers.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <!-- #region template -->
+  <div class="d-flex gap-2">
+    <BButton id="popover-target-default-trigger"> Default Trigger </BButton>
+    <BPopover target="popover-target-default-trigger" placement="top" body="Default Trigger" />
+    <BButton id="popover-target-click-trigger"> Click Trigger </BButton>
+    <BPopover target="popover-target-click-trigger" click placement="top" body="Click Trigger" />
```

Comment:
```suggestion
    <BPopover body="Default Trigger">
      <template #target>
        <BButton> Default Trigger </BButton>
      </template>
    </BPopover>
    <BPopover click body="Click Trigger">
      <template #target>
        <BButton> Click Trigger </BButton>
      </template>
    </BPopover>
```

I'd prefer the examples syntax to be "narrow" so that it's more readable on mobile

Reply:
I'm using prettier to format - I think I'm using the settings suggested by the project. I can look at overriding the line length in prettier for the demo directories (I think that's probably possible) - Do you have a suggestion on line length for the samples?